docusaurus/docs/Android/03-ui/04-components/01-channel-list.md

```diff
@@ -67,17 +126,21 @@ TransformStyle.channelListStyleTransformer = StyleTransformer { defaultStyle ->
     )
 }
 ```
+
+These changes should have the following results:
+
+| Before | After |
+| --- | --- |
+|![before](../../assets/channel_list_view_style_before.png)|![after](../../assets/channel_list_view_style_after.png)|
```

Comment:
At some point in the nearish future, we should look into a set of standards for how we add alt-text to these before and after images. Partly it would be to make the website friendlier to screen readers but also partly because having alt text explain what the images are demonstrating should make the page easier for google to parse.